scipy/stats/stats.py

```diff
@@ -1793,18 +1800,41 @@ def cumfreq(a, numbins=10, defaultreallimits=None, weights=None):
 
     Examples
     --------
+    >>> import matplotlib.pyplot as plt
     >>> from scipy import stats
     >>> x = [1, 4, 2, 1, 3, 1]
-    >>> cumfreqs, lowlim, binsize, extrapoints = stats.cumfreq(x, numbins=4)
-    >>> cumfreqs
-    array([ 3.,  4.,  5.,  6.])
-    >>> cumfreqs, lowlim, binsize, extrapoints = \
-    ...     stats.cumfreq(x, numbins=4, defaultreallimits=(1.5, 5))
-    >>> cumfreqs
+    >>> res = stats.cumfreq(x, numbins=4, defaultreallimits=(1.5, 5))
+    >>> res.cumcount
     array([ 1.,  2.,  3.,  3.])
-    >>> extrapoints
+    >>> res.extrapoints
     3
 
+    Create a normal distribution with 1000 random values
+
+    >>> rng = np.random.RandomState(seed=12345)
+    >>> samples = stats.norm.rvs(size=1000, random_state=rng)
+
+    Calculate cumulative frequencies
+
+    >>> res = stats.cumfreq(samples, numbins=25)
+
+    Calculate space of values for x
+
+    >>> x = res.lowerlimit + np.linspace(0, res.binsize*res.cumcount.size,
+                                         res.cumcount.size)
+
+    Plot 'Histogram' and 'Cumulative histogram'
```

Comment:
This needs a blank line below it to render correctly. Also I would format it as ``Plot histogram and cumulative histogram:``